cpp/include/cugraph/dendrogram.hpp

```diff
@@ -15,7 +15,9 @@
  */
 #pragma once
 
+#include <rmm/cuda_stream_view.hpp>
 #include <rmm/device_uvector.hpp>
+#include <rmm/device_vector.hpp>
```

Comment:
Why would this be needed?

Reply:
You're right. I moved the include statement to the relevant files.